django_netjsongraph/templates/netjsongraph/netjsongraph-script.html

```diff
@@ -1,12 +1,27 @@
 <script>
-    var graph = d3.netJsonGraph("{{ api_url }}", {
-        linkClassProperty: "status",
-        defaultStyle: false,
-        labelDy: "-1.5em",
-        circleRadius: 7,
-        charge: -300,
-        gravity: 0.1,
-        linkDistance: 70,
-        linkStrength: 0.1
-    });
+    // the graph may be loaded elsewhere than body
+    window.__njg_el__ = window.__njg_el__ || "body";
```

Comment:
This file is amazing! Didn't know we could do all these :open_mouth:

Reply:
probably not optimal because not very easy to read, but a more elegant solution would require more time.
we basically reuse this piece of code to load and reload the graph in frontend, admin and when loading older snapshots, the difference is that in the admin we set `window.__njg_el__` to the overlay, and when using snapshots we first fetch the data from the API and then pass the data in the graph directly to the library.

Reply:
Yeah. I understand. I really liked how you made it reusable